utils/rowgroup/rowgroup.h

```diff
@@ -339,7 +339,7 @@ class Row
     inline Pointer() = default;
 
     // Pointer(uint8_t*) implicitly makes old code compatible with the string table impl;
-    inline Pointer(uint8_t* d) : data(d)
+    explicit inline Pointer(uint8_t* d) : data(d)
```

Comment:
Row::setData() function declaration at line 364 below has the comment: "convenience fcn, can go away". Can you please remove this comment since this function is clearly used at other places in the code.

Reply:
++